xtask/src/testing.rs

```diff
@@ -0,0 +1,45 @@
+//
+// Copyright 2023 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+use crate::internal::{read_to_end, Command, Context, Opt, Runnable, Running, Scope, Status, Step};
+
+fn opt_for_test() -> Opt {
+    Opt {
+        dry_run: false,
+        logs: true,
+        keep_going: false,
+        scope: Scope::All,
+        cmd: Command::RunTests,
+    }
+}
+
+/// Runs a step, and returns the result.
```

Comment:
```suggestion
/// Runs a step, and checks that its execution is successful.
```

Reply:
Thanks, that was from a previous version